packages/oauth/examples/oauth-v1/README.md

```diff
@@ -0,0 +1,52 @@
+# OAuth v1 Example
+
+This repo contains a sample app for doing OAuth with Slack for Classic Slack Apps. I recommending checking out `app.js`. I've included a few different options in the code which have been commented out. As you play around with the app, I recommend you uncomment some of these options to get a deeper understanding of how to use this library. 
+
+I recommend using [`ngrok`](https://ngrok.com/download) for local development of Slack apps. Checkout [this guide](https://api.slack.com/tutorials/tunneling-with-ngrok) for setting it up.
```

Comment:
Local development requires a public URL where Slack can send requests. In this guide, we'll be using [`ngrok`](https://ngrok.com/download). Checkout [this guide](https://api.slack.com/tutorials/tunneling-with-ngrok) for setting it up.